### DISCUSSION TITLE
brush supported formats

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#14] [Q&A]
hihi, do you have any other supported brush formats in mind ? :] or just photoshop ? also do you plan to do this for android and ios phone users ? :0 or just apple only ?

**Comment by keiwando** [comment]
For now the next step would be adding dual brush support for the ToonSquid and Photoshop brushes, but feel free to mention other brush formats that you are interested in.

And the app itself is going to be iPad-only. M1 Mac support would be an option if enough people ask for it but it would need to be a lot of people.

**Reply by AQuickSecond** [reply]
A way to port Procreate brushes and brushsets over would be very convenient

**Reply by ivana67dj** [reply]
+++ that would be incredibly convenient
